go.mod

```diff
@@ -33,6 +32,7 @@ require (
 	github.com/spf13/viper v1.8.0
 	github.com/tektoncd/pipeline v0.25.0
 	github.com/tektoncd/triggers v0.14.2
+	github.com/vifraa/gopom v0.1.0 // indirect
```

Comment:
I assume we are not using this rather your own version. So we can remove this reference.